packages/frontend/src/features/allertsPanel/components/AllertCell.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react";
+import styled from "styled-components";
+
+import { FontSize } from "../../common/styles/constans/FontSize";
+import { FontWeight } from "../../common/styles/constans/FontWeight";
+import { Device } from "../../common/styles/constans/Device";
+
+const AllertContainer = styled.div`
+  position: relative;
+  display: flex;
+  width: 100%;
+  min-height: 77px;
+  border-radius: 10px;
+  box-shadow: 0px 3px 6px 0px #00000029;
+  margin-bottom: 25px;
+  padding: 5px 10px;
+  border: 2px solid transparent;
```

Comment:
po co Ci niewidoczny border?

Reply:
niewidoczny border jest po to żeby w momencie kiedy element jest aktywny nierozjeżdżał się